### PR TITLE
Kintwin controller for kinesis keyboard, split layout

### PR DESCRIPTION
**Pull request by alvicstep**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kinesis/kintwin/config.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* Copyright 2023 Alex Stepanov
```
Comment (fauxpark) [inline keyboards/kinesis/kintwin/config.h:1]: This file can be deleted.

File: keyboards/kinesis/keymaps/default/keymap.c [hunk 2]
```diff
@@ -1,3 +1,19 @@
+/* Copyright 2023 QMK
```
Comment (fauxpark) [inline keyboards/kinesis/keymaps/default/keymap.c:1]: Prefer SPDX headers eg.
```c
// Copyright %YEAR% %REAL_NAME% (@%USER_NAME%)
// SPDX-License-Identifier: GPL-2.0-or-later
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kinesis/kintwin/info.json [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        "shared_endpoint": {
+            "keyboard": true   
+        } 
+    },
+    "features": {
+        "bootmagic": true,
+        "console": false,
+        "command": false,
+        "mousekey": false,
+        "extrakey": false,
```
Comment (zvecr) [inline keyboards/kinesis/kintwin/info.json:20]:
```suggestion
        "mousekey": true,
        "extrakey": true,
```

File: keyboards/kinesis/kintwin/halconf.h
Comment (zvecr) [inline keyboards/kinesis/kintwin/halconf.h:1]: This file should be removed as nothing is configured to use pwm or i2c.

File: keyboards/kinesis/kintwin/mcuconf.h
Comment (zvecr) [inline keyboards/kinesis/kintwin/mcuconf.h:1]: This file should be removed as nothing is configured to use pwm or i2c.

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kinesis/rules.mk
Comment (zvecr) [inline keyboards/kinesis/rules.mk:1]: Removing this file, and not migrating its settings to each build revision will mean they now have the various features disabled now. The `features` object should be added to the relevant `info.json` files.

File: keyboards/kinesis/kintwin/info.json [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+                {"matrix": [6, 9], "x": 8.25, "y": 6},
+                {"matrix": [5, 8], "x": 9.25, "y": 6},
+
+                {"matrix": [5, 3], "x": 6.25, "y": 7},
+
+                {"matrix": [6, 8], "x": 8.25, "y": 7},
+
+                {"matrix": [6, 3], "x": 4.25, "y": 7, "h": 2},
+                {"matrix": [6, 4], "x": 5.25, "y": 7, "h": 2},
```
Comment (fauxpark) [inline keyboards/kinesis/kintwin/info.json:137]:
```suggestion
                {"matrix": [6, 4], "x": 5.25, "y": 7, "h": 2},
```
etc

**Review by fauxpark — APPROVED** [review]

File: keyboards/kinesis/kintwin/info.json [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+    "keyboard_name": "Kinesis",
+    "url": "https://github.com/alvicstep/kintwin",
+    "manufacturer": "https://github.com/alvicstep/kintwin",
+    "maintainer": "Alex Stepanov",
+    "usb": {
+        "device_version": "0.0.3",
+        "vid": "0xFEED",
+        "pid": "0x4B4E",
+        "shared_endpoint": {
+            "keyboard": true
```
Comment (fauxpark) [inline keyboards/kinesis/kintwin/info.json:12]: etc

**Comment by laffed** [comment]
@zvecr  please <3
7v4w9e

**Review by drashna — APPROVED** [review]

**Comment by asegarra** [comment]
@alvicstep apologize for contacting you here but found no other way, just wanted to know if your board is compatible with STM32F411 ?

**Comment by alvicstep** [comment]
Yes it is

It is a fork of kinbp board

You can find github pages for both boards

Thanks

**Pull request merged**